Cython/Plex/Transitions.py

```diff
@@ -84,15 +80,13 @@ def add_set(self, event, new_set,
         else:
             self.get_special(event).update(new_set)
 
-    def get_epsilon(self,
-                    none=None):
+    def get_epsilon(self, none=None):
```

Comment:
`None` is a constant now, so this is a useless (de-)optimisation. Actually, in case `self.special` is an actual `dict`, it is even entirely unnecessary to pass the argument into `.get()` at all.

Reply:
This is only ever called in one place and no kwarg is pased.  So just remove the `none` kwarg altogether?

Reply:
Yes, it can be removed. `self.special` is a `dict`, I checked.